components/atom/tooltip/src/hoc/withIntersectionObserver.js

```diff
@@ -0,0 +1,39 @@
+/* global IntersectionObserver */
+import React, {Component} from 'react'
+import 'intersection-observer' // polyfill
```

Comment:
I think this polyfill is not isomorphic, thus this component will break server side rendering. We could do something like this: https://github.com/miduga/react-slidy/blob/master/src/index.js#L7

Reply:
Is there a way I can try the tooltip demo in the playground using server side rendering? Or any other documented way i can check this possible issue?

Reply:
Added SSR support in this HOC. Tested using this quick-boilerplate-for-SSR → https://github.com/juanmaguitar/quick-ssr-test-sui-components